src/shell.js

```diff
@@ -105,7 +105,7 @@ if (ENVIRONMENT_IS_NODE) {
   // We need to use `createRequire()` to construct the require()` function.
   const { createRequire } = await import('module');
   /** @suppress{duplicate} */
-  var require = createRequire('/');
+  var require = createRequire(import.meta.url);
```

Comment:
I don't know enough about support for this - didn't we have an issue with it breaking in some bundler or environment?

Reply:
I believe you are thinking of the use of `import.meta.url` of in `new Worker`.   This usage is perfectly fine AFAIK.  See #23169 which removed it prematurely.